### ISSUE TITLE
"Why did I join source{d}" the blog series

### ISSUE THREAD

**Issue by campoy** [src-d/blog#175]
Each new employee should write a "why did join source{d}" as proposed by @bzz on https://github.com/src-d/devrel/issues/5

**Comment by vmarkovtsev** [comment]
Shall every former employee write "why did leave source{d}" as well? :trollface: @smola @serabe

**Comment by campoy** [comment]
I'm not against it! After all I wrote a "why I left Google" so ...

**Comment by bzz** [comment]
This sounds great! 

To kick-start this effort, @campoy do you think it can be possible to cross-post your post first? Then, I believe, @smacker and @carlosms might be encouraged enough to follow your steps.

**Comment by campoy** [comment]
Absolutely, I'll send a PR with the content. It was already written in markdown for campoy.cat 😄

**Comment by campoy** [comment]
Update: https://github.com/src-d/blog/pull/177

**Comment by Serabe** [comment]
I have no problem writing one when I find some time.

**Comment by vcoisne** [comment]
Love this idea any interest @creachadair @Jan21 @aliciakozlowski @m09 ?

**Comment by creachadair** [comment]
I'd be willing to write something like this, though I'm not sure my perspective would be interesting to anyone outside the company.

**Comment by meyskens** [comment]
Happy to write something (small), however would love some feedback as I'm not a professional blog writer.

**Comment by Jan21** [comment]
I could also write something at the end of my internship.

**Comment by vcoisne** [comment]
I think that would be great @creachadair @meyskens and @Jan21. Keep it short and sweet, no need to write more than 500 words

**Comment by m09** [comment]
Same for me I'm up for writing a short text.

**Comment by vcoisne** [comment]
> I'd be willing to write something like this, though I'm not sure my perspective would be interesting to anyone outside the company.

I agree with you, I think this would mostly help @bluer73 - What do you think Robbie ?

**Comment by bluer73** [comment]
@vcoisne really keen on this idea- would be great to get one person from each engineering team- it would also be great as a "blog" rather than just Q&A but we could have 4/5 key questions to structure it.
